tests/Feature/ReadThreadsTest.php

```diff
@@ -107,4 +108,20 @@ function we_record_a_new_visit_each_time_the_thread_is_read()
 
         $this->assertEquals(1, $thread->fresh()->visits);
     }
+
+    /** @test */
+    function requests_from_robots_are_not_recorded_as_new_visits()
+    {
+        // The user-agent of the Google web crawling bot as specified at:
+        // https://support.google.com/webmasters/answer/1061943?hl=en
+        Agent::setUserAgent('Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)');
+
+        $thread = create('App\Thread');
+
+        $this->assertSame(0, $thread->visits);
```

Comment:
Why `assertSame` here, but `assertEquals` later?

Reply:
Good question. I just copied the previous test ` we_record_a_new_visit_each_time_the_thread_is_read()` and modified it. I thought the same. 

I can change them if appropriate?